zstacklib/zstacklib/utils/ovn.py

```
@@ -0,0 +1,50 @@
+'''
+
+@author: haibiao.xiao
+'''
+import os
```

Ruff (0.8.2)

`os` imported but unused

Remove unused import: `os`

(F401)